tools/replicate_raisinbread_for_mcin_dev_vm.pl

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/perl -w
+
+=pod
+
+=head1 NAME
+
+replicate_raisinbread_for_mcin_dev_vm.pl -- Create a local copy of the RaisinBread dataset and
+store each file as a symlink to the original dataset found in /data-raisinbread automatically mounted
+with all LORIS dev VMs created by MCIN. 
+
+=head1 SYNOPSIS
+
+perl replicate_raisinbread_for_mcin_dev_vm.pl C<[/path/to/mounted/raisinbread]> C<[/path/to/output_dir]> 
+
+=head1 DESCRIPTION
+
+This script takes in two arguments. The first argument is the path to the
+RaisinBread dataset (typically C</data-raisinbread>) and the path to the
+directory where the replicated dataset will be stored. For example, suppose
+the script is run with the following arguments:
+
+perl replicate_raisinbread_for_mcin_dev_vm.pl C</data-raisinbread> C</data>
+
+The replicated dataset will be found in C</data/data-raisinbread/> 
+
+=head1 LICENSING
+
+License: GPLv3
+
+=head1 AUTHORS
+
+LORIS community <loris.info@mcin.ca> and McGill Centre for Integrative Neuroscience
+
+=cut
+
+use strict;
+use warnings;
+use NeuroDB::ExitCodes;
+
+use Cwd qw/abs_path/;
+use File::Find;
+use File::Spec::Functions qw/catfile abs2rel/;
+use File::Path qw/make_path/;
+
+my $Usage = <<USAGE;
+usage:  $0 /path/to/raisinbread /path/to/output_dir
+
+This script creates a local copy of the RaisinBread dataset and stores each file 
+as a symlink to the original dataset.
+
+This script takes in two arguments. The first argument is the path to the
+RaisinBread dataset (typically /data-raisinbread) and the path to the
+directory where the replicated dataset will be stored. For example, suppose
+the script is run with the following arguments:
+
+perl replicate_raisinbread_for_mcin_dev_vm.pl /data-raisinbread /data/data-raisinbread/
+
+The replicated dataset will be found in /data/data-raisinbread/
+
+USAGE
+
+# Checks if the right number of arguments are given
+if (scalar(@ARGV) != 2) {
+    print "ERROR: Incorrect number of arguments\n";
+    print $Usage;
+    exit $NeuroDB::ExitCodes::MISSING_ARG;
+}
+
+# Get the path of RaisinBread dataset and the directory where the copy dataset 
+# will be stored
+my $rb_dir      = abs_path($ARGV[0]);
+my $output_dir  = abs_path($ARGV[1]);   
```

Comment:
sorry, should have thought about this in my initial review... Here, it might be good to check that the directories exists on the file system before continuing. It they do not, issue an error and exit.

If the paths exist, then it might be good to print in the terminal some messages:
"Replicating the data structure from $rb_dir" to $output_dir and linking files"

Reply:
I added a check for the validity of the input paths and I added helpful messages for the user. Let me know if there is anything else